tox.ini

```diff
@@ -56,15 +53,6 @@ deps =
     {env:_PYTEST_TOX_EXTRA_DEP:}
 platform = {env:_PYTEST_TOX_PLATFORM:.*}
 
-[testenv:py27-subprocess]
-deps =
-    pytest-xdist>=1.13
-    py27: mock
-    nose
-commands =
-    pytest -n auto --runpytest=subprocess {posargs}
```

Comment:
Should this move into py37-subprocess then?
(But I'd be OK with removing it)

Reply:
I'm fine removing it too.